airflow/providers/databricks/utils/databricks.py

```diff
@@ -17,6 +17,9 @@
 # under the License.
```

Comment:
@alexott FYI: The `Escaper` class was not available on the `main` branch. So, I added it as part of this PR to be used in our provider. 
Also, made some minor changes to exception handling in the Escaper class because it was looking for some user defined Exception classes from `pyhive`.

Reply:
Hmmm, it should be a part of `databricks-sql-connector`: https://github.com/databricks/databricks-sql-python/blob/main/src/databricks/sql/utils.py#L121

Reply:
How do I use the Escaper class in databricks-sql-python within Airflow? Do I add it to setup.py to be installed when Airflow starts?

Reply:
Thanks for the help, I imported it via `from databricks.sql.utils import ParamEscaper`